.github/workflows/main.yml

```diff
@@ -17,12 +19,43 @@ jobs:
       BUILD_VERSION: ${{ github.ref_name }}
     steps:
     - uses: actions/checkout@master
-    - run: echo "$BUILD_REF $BUILD_VERSION"
     - name: Prepare
       run: |
-        mkdir tmp
+        mkdir _tmp
         ./.docker/docker.sh prepare
     - name: Build docker images
       run: ./.docker/docker.sh build
     - name: Test docker images
       run: ./.docker/docker.sh test
+  release:
+    runs-on: ubuntu-20.04
+    if: ${{ github.ref_type == 'tag' }}
+    env:
+      BUILD_REF: ${{ github.sha }}
+      BUILD_VERSION: ${{ github.ref_name }}
+    steps:
+    - uses: actions/checkout@master
+    - name: Prepare
+      run: |
+        mkdir _tmp
+        ./.docker/docker.sh prepare
+    - name: Login to Docker Hub
+      uses: docker/login-action@v1
```

Comment:
Leverages https://github.com/docker/login-action

and will need credentials in the secrets to work

Reply:
@reloxx13 for this to work we need to include the dockerhub username/token in the repo secrets